Jupyter_Notebooks/PreProcess.py

```diff
@@ -0,0 +1,279 @@
+import ipywidgets as widgets
+import numpy as np
+import sys
+import scipy.io
+sys.path.insert(1, 'C:/Users/joelb/pyCUFSM')
```

Comment:
I'm not super familiar with Jupyter Notebook's programming specifically, but there definitely shouldn't be paths pointing to your personal local computer committed onto GitHub!

Reply:
Yes..sorry the Jupyter Notebook files weren't ready to commit yet(added them to my repo to test the notebooks on Colab)..And this particular script isn't gonna be in the final version..
Thought I'd pull request on some of the changes(plotters.py) and once they were reviewed would work on adding the rest of the files

Reply:
Yes, committing in small chunks makes it faster and easier for me to review too :-)  You could either edit your branch here, or cancel this pull request and create a new one of a more limited branch.